README.md

```diff
@@ -32,6 +32,34 @@ To create a new version of the documentation:
 
 ## Contributing to docs
 
+### Sign Off on All Commits
+
+All contributions to the docs need to be signed off.
+
+To sign off when creating a commit, run:
+
+    git commit --signoff
+
+To sign off when editing the docs with the GitHub UI, enter a name for your commit, then in the large field below the commit message, enter the signoff text with your own name and email:
+
+    Signed-off-by: Catherine Luse <catherine.luse@rancher.com>
+
+To sign off on a commit that is already in a pull request, 
+
+1. Head to your local branch and run:
+
+    git commit --amend --signoff
+
+    Now your commits will have your signoff. 
+    
+2. Next run:
+
+    git push --force-with-lease origin patch-1
```

Comment:
`patch-1` is a local branch right? Probably want to explain that.

Reply:
updated